### PR TITLE
Splitting post jobs for waves

### PR DESCRIPTION
**Pull request by JessicaMeixner-NOAA**
This adds two additional wave jobs for point processing at the end of the job and then for boundary point jobs removing the points from the side-by-side jobs.   The boundary point job supplies new output needed for downstream jobs and requires a change in fix files.  There will be another change for optimizing these jobs as a follow up to this. 

This also adds the cdo module used in wave prep and small changes related to using this module in ush/wave_prnc_cur.sh
There will be no fix-file changes related to this. 

There are a few clean up/ changes for better error handling. 

Changes were made for several of the resources for wave jobs. There should be no impact in performance. 

Testing the forecast only reproduces the real-time parallel for atm and has the expected changes for waves.

### CODE REVIEW

**Comment by KateFriedman-NOAA** [comment]
I will be reviewing the changes and testing them against v16rt2 on WCOSS-dev before approving.

File: jobs/JGLOBAL_WAVE_POST_BNDPNT [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+export COMOUT=${COMOUT:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/$COMPONENT}
+
+export COMINice=${COMINice:-${COMROOTp2}/omb/prod}
+export COMINwnd=${COMINwnd:-${COMROOT}/gfs/prod}
+export COMIN_WAV_CUR=${COMIN_WAV_CUR:-${COMROOTp2}/rtofs/prod}
+
+mkdir -p $COMOUT/station
+
+# Set mpi serial command
+export wavempexec=${launcher:-"mpirun -n"}
```
Comment (yangfanglin) [inline jobs/JGLOBAL_WAVE_POST_BNDPNT]: These should be defined in  the ./env/WCOSS_DELL_P3.env file for WCOSS, and for others platforms as well

File: jobs/JGLOBAL_WAVE_POST_PNT [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+export COMINice=${COMINice:-${COMROOTp2}/omb/prod}
+export COMINwnd=${COMINwnd:-${COMROOT}/gfs/prod}
+export COMIN_WAV_CUR=${COMIN_WAV_CUR:-${COMROOTp2}/rtofs/prod}
+
+mkdir -p $COMOUT/station
+
+# Set mpi serial command
+export wavempexec=${launcher:-"mpirun -n"}
+export wave_mpmd=${mpmd:-"cfp"}
```
Comment (yangfanglin) [inline jobs/JGLOBAL_WAVE_POST_PNT]: define in  files under ./env

File: jobs/JGLOBAL_WAVE_POST_BNDPNT (hunk 1 above)
Comment (JessicaMeixner-NOAA) [reply jobs/JGLOBAL_WAVE_POST_BNDPNT]: launcher is defined in the ./env/*.env files.  Do you want the definition of wavempexec also set in env or the default value removed?

File: jobs/JGLOBAL_WAVE_POST_PNT (hunk 2 above)
Comment (JessicaMeixner-NOAA) [reply jobs/JGLOBAL_WAVE_POST_PNT]: launcher and mpmd are defined in env files.  Should wavempexec/wave_mpmd  also defined there?

Comment (yangfanglin) [reply jobs/JGLOBAL_WAVE_POST_PNT]: We do define these type of commands in ./env using, for instance,  APRUN_WAVE,  APRUN_CFP etc for different platforms

Comment (JessicaMeixner-NOAA) [reply jobs/JGLOBAL_WAVE_POST_PNT]: done.

File: jobs/JGLOBAL_WAVE_POST_BNDPNT (hunk 1 above)
Comment (JessicaMeixner-NOAA) [reply jobs/JGLOBAL_WAVE_POST_BNDPNT]: done.

**Review by KateFriedman-NOAA — APPROVED** [review]
Ran updates successfully in test on Mars. Jessica checked output and confirmed jobs ran correctly. Also tested gempak updates from Roberto in the same test and he confirmed those updates ran correctly and that folks at OPC will now examine gempak output once the updates are in the real-time parallel.

**Pull request merged**